packages/editor/web/src/RichContentEditor/InnerRCE.jsx

```diff
@@ -24,11 +24,19 @@ class InnerRCE extends Component {
     this.setState({ MobileToolbar, TextToolbar });
   }
 
+  static getDerivedStateFromProps(props, state) {
+    return !state?.editorState?.getSelection?.().hasFocus ? { editorState: props.editorState } : {};
```

Comment:
Are you sure they are needed. I think you always have editorState?
```suggestion
    return !state.editorState.getSelection.().hasFocus ? { editorState: props.editorState } : {};
```